Index.html

```diff
@@ -186,5 +186,253 @@ <h2 class="hero-text-title">I'm a Full Stack Web Developer</h2>
+          <div class="recent-works-skills-box">
+            <span class="recent-works-col-skills">HTML</span>
+            <span class="recent-works-col-skills">CSS</span>
+            <span class="recent-works-col-skills">RAILS</span>
+            <a class="card-button" href="https://jose-abel-finance-tracker.herokuapp.com">See Project</a>
+          </div>
+        </div>
+
+        <div class="recent-works-col recent-works-col-4">
+          <h3 class="recent-works-col-title">Telegram Bot</h3>
+          <p class="recent-works-col-text">A Telegram Bot that connects to the coinmarketcap API to get the latest price for the top 10 cryptos and connects to cryptopanic.com to get the latest news.</p>
+          <div class="recent-works-skills-box">
+            <span class="recent-works-col-skills">RUBY</span>
+            <a class="card-button" href="https://t.me/Price10CryptoBot">See Project</a>
+          </div>
+        </div>
+
+        <div class="recent-works-col recent-works-col-5">
+          <h3 class="recent-works-col-title-5">Mapty Entertainment Tracker</h3>
+          <p class="recent-works-col-text">Website that connects to the geolocation API and with the leaflet library display a map geolocated to your address. </p>
+          <div class="recent-works-skills-box">
+            <span class="recent-works-col-skills">HTML</span>
+            <span class="recent-works-col-skills">CSS</span>
+            <span class="recent-works-col-skills">JAVASCRIPT</span>
+            <a class="card-button" href="https://jose-abel-entertainment-mapty-tracker.netlify.app/">See Project</a>
+          </div>
+        </div>
+
+        <div class="recent-works-col recent-works-col-6">
+          <h3 class="recent-works-col-title">Conference Page</h3>
+          <p class="recent-works-col-text">A conference page website, created with 3 different webpages, which are: the main page, the about page and the tickets sales page.</p>
+          <div class="recent-works-skills-box">
+            <span class="recent-works-col-skills">HTML</span>
+            <span class="recent-works-col-skills">CSS</span>
+            <a class="card-button" href="https://jose-abel-conference-page.netlify.app/">See Project</a>
+          </div>
+        </div>
+
+      </div>
+    </section>
+
+    <section class="about-me-section" id="about-me-section-id">
+
+      <div class="about-me-section-box">
+        <div class="about-section-text-box">
+          <h1 class="about-text-h1">About me</h1>
+          <p class="about-text-p">Hello I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me</p>
+          <a class="get-resume-btn" href="./docs/Jose-Abel-Ramirez-Frontany.pdf" download>Get My Resume</a>
+        </div>
```

Comment:
- [ ] Please make sure the about section matches with the Figma design.
    - Please use the correct font-size, font-weight, and font-family.
    - Please fix the background image.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166122964-5f545044-d65c-4b74-87bc-d1647e9734cf.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166122972-f4902dde-f275-490e-8901-c9bd43f02d8b.png)